velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp

```diff
@@ -90,7 +91,9 @@ std::unique_ptr<ReadFile> HdfsFileSystem::openFileForRead(
 
 std::unique_ptr<WriteFile> HdfsFileSystem::openFileForWrite(
     std::string_view path) {
-  VELOX_UNSUPPORTED("Write to HDFS is unsupported");
+  std::string path_str(path);
```

Comment:
naming: path_str -> pathStr

Please, take a moment to review coding style and update the PR to comply: https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#naming-conventions